### PR TITLE
[_]: feat/hiding or displaying options depending on the role

### PR DESCRIPTION
**Pull request by rafijv**
- Reload list after manage users access
- Temporarily hide general access public option
- Hide general access options for users without permissions
- Hide invite button to viewers
- Temporarily Hide copy link button

### CODE REVIEW

File: src/app/share/views/SharedLinksView/SharedView.tsx [hunk 1]
```diff
@@ -108,6 +108,18 @@ export default function SharedView(): JSX.Element {
     }
   }, []);
 
+  useEffect(() => {
+    if (!isShowInvitationsOpen) fetchRootItems();
+  }, [isShowInvitationsOpen]);
```
Comment (CandelR) [inline src/app/share/views/SharedLinksView/SharedView.tsx]: I think that this has been removed in the accept/decline invitations task.

File: src/app/drive/components/ShareDialog/ShareDialog.tsx [hunk 2]
```diff
@@ -133,6 +134,11 @@ const ShareDialog = (props: ShareDialogProps): JSX.Element => {
     return () => clearTimeout(timer);
   }, [accessRequests]);
 
+  useEffect(() => {
+    const currentInvitedUser = invitedUsers.find((user) => user.email === props.user.email);
+    setCurrentUserFolderRole(currentInvitedUser?.roleName);
+  }, [invitedUsers]);
```
Comment (CandelR) [inline src/app/drive/components/ShareDialog/ShareDialog.tsx:140]: When is no user invited, at the moment it not appear any user on the list. This causes that the owner couldn't invite users with this logic.

**Review by CandelR — CHANGES_REQUESTED** [review]

File: src/app/share/views/SharedLinksView/SharedView.tsx (hunk 1 above)
Comment (rafijv) [reply src/app/share/views/SharedLinksView/SharedView.tsx]: Removed, was an error merging branches

File: src/app/drive/components/ShareDialog/ShareDialog.tsx (hunk 2 above)
Comment (rafijv) [reply src/app/drive/components/ShareDialog/ShareDialog.tsx:140]: Fixed

**Review by CandelR — APPROVED** [review]

**Pull request merged**